sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -0,0 +1,90 @@
+import { Controller, Get, Param } from "@nestjs/common";
+import BaseController from "../BaseController";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { ClientTypeBaseRoute } from "../../types";
+import { ApiOkResponse, ApiTags } from "@nestjs/swagger";
+import { UserGroups } from "../../auth/user-groups.enum";
+import {
+  StudentAppealService,
+  StudentAssessmentService,
+  StudentScholasticStandingsService,
+} from "../../services";
+import { AssessmentTriggerType } from "../../database/entities";
+import { RequestAssessmentSummaryDTO } from "./models/assessment.dto";
+import { AssessmentHistory } from "src/services/student-assessment/student-assessment.models";
+
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("assessment")
+@ApiTags(`${ClientTypeBaseRoute.AEST}-assessment`)
+export class AssessmentAESTController extends BaseController {
+  constructor(
+    private readonly studentAppealService: StudentAppealService,
+    private readonly studentScholasticStandingsService: StudentScholasticStandingsService,
+    private readonly studentAssessmentService: StudentAssessmentService,
+  ) {
+    super();
+  }
+
+  /**
+   * Controller to get all requested assessments for an student
+   * application, i.e, this will fetch the combination of
+   * pending and denied student appeal and scholastic
+   * standings for an application.
+   * @param applicationId, application number.
+   * @returns RequestAssessmentSummaryDTO list.
+   */
+  @Get("application/:applicationId/requests")
+  @ApiOkResponse({
+    description: "Requested assessment for the application found.",
+  })
+  async getRequestedAssessmentSummary(
+    @Param("applicationId") applicationId: number,
+  ): Promise<RequestAssessmentSummaryDTO[]> {
+    const [studentAppealQuery, studentScholasticStandingsQuery] =
```

Comment:
The response variables can be renamed more relevantly.

Reply:
I did not follow why the variables could be considered "not relevantly named".
I saw that they were already renamed, but both ways would work, IMO.